apps/1d_stencil/stencil_stdexec.cpp

```diff
@@ -121,12 +128,14 @@ int benchmark(args_params_t const& args) {
             case sch_t::CPU:
                 solution = step.do_work(exec::static_thread_pool(nthreads).get_scheduler(), size, nt);
                 break;
+#if defined(GPUSTDPAR)
```

Comment:
I wonder if we can get rid of this type of macros as they are intrusive to the codebase and we have to maintain the macros.

Reply:
done. renamed to `USE_GPU`